fairlearn/reductions/__init__.py

```diff
@@ -18,15 +18,7 @@
 from ._moments import BoundedGroupLoss, SquareLoss, ZeroOneLoss  # noqa: F401
 from ._moments import ClassificationMoment, LossMoment  # noqa: F401
 
-_exponentiated_gradient = [
```

Comment:
This seems.... unrelated?

Reply:
Flake8 is requiring this to change. RST902 failure and I can't override it with `noqa` it seems. So I played around with it and this is the only solution I can come up with. This is probably hitting every PR right now (?)

Reply:
Moving this a different PR